File: BahamasSystem/BahamasSystem/RiskManager.cpp
#include "RiskManager.h"

RiskManager::RiskManager()
{
}


RiskManager::~RiskManager()
{
}

std::vector<MarketOrder> RiskManager::ProfileOrder(MarketOrder& order) {
	std::vector<MarketOrder> results;
	results.push_back(order);
	
	return results;
}

File: BahamasEngine/BahamasEngine/RiskManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BahamasEngine
{
    public class RiskManager
    {
        public RiskManager()
        {

        }

        public IList<MarketOrder> ProfileOrder(MarketOrder order)
        {
            IList<MarketOrder> orders = new List<MarketOrder> { order };
            return orders;
        }
    }
}
